CHANGELOG.md

```diff
@@ -642,6 +642,15 @@ Anyway, the cluster should get back to the proper size after 10 min.
          the new CRI + CNI code path.
       * If you are using plugins other than `bridge`, make sure you have
         updated custom plugins to the latest version that is compatible.
+* **CNI plugins now affect node readiness**
+  * Kubelet will now block node readiness until a CNI configuration file is
+    present in /etc/cni/net.d or a given custom CNI configuration path.
+    [This change](https://github.com/kubernetes/kubernetes/pull/43474) ensures
+    kubelet does not start pods that require networking before
+    [networking is ready](https://github.com/kubernetes/kubernetes/issues/43014).
+    This change may require changes to clients if the client gates pod scheduling
```

Comment:
I'm okay with the text as is, but right now no pods that go through the regular scheduling path (i.e. all except DaemonSet) will land on the machine.  This is a break in behavior and is the reason we had to remove the "test deployment" check from kubeadm.

So -- if someone relied on pods being scheduled and working (with host networking) that is now broken and is no longer possible.  The scheduler doesn't have a carve out for host networking pods on nodes with broken networking.

Previously non host-networking pods wouldn't launch but would be scheduled.  Now they won't be scheduled at all.  So they are now a different type of broken.

This is all pretty subtle and I doubt that many folks actually were relying on host networking pods working with broken CNI, but you never know.